common.yml

```diff
@@ -5,15 +5,15 @@ services:
       context: ./src/django
       dockerfile: Dockerfile
     environment:
-      - PGHOST=database.service.pfb.internal
-      - PGDATABASE=pfb
-      - PGPASSWORD=pfb
-      - PGPORT=5432
-      - PGUSER=pfb
       - DJANGO_ENV=development
-      - DJANGO_SECRET_KEY=secret
       - DJANGO_LOG_LEVEL=INFO
       - AWS_PROFILE=pfb
+      - PFB_DB_HOST=database.service.pfb.internal
```

Comment:
Maybe I missed it among all the other changes, but these changes should also be reflected in the ECS task definition files in Terraform: 

https://github.com/azavea/pfb-network-connectivity/tree/develop/deployment/terraform/task-definitions